src/NexusMods.App.UI/Resources/Language.pl.resx

```diff
@@ -261,6 +261,15 @@
 	<data name="SpineHomeButton_ToolTip_Home" xml:space="preserve">
 		<value>Strona główna</value>
 	</data>
+    <data name="LoadoutGridView__View_Files" xml:space="preserve">
+        <value>Zobacz Pliki Moda</value>
```

Comment:
It should be `Zobacz pliki modyfikacji` or optionally `Zobacz pliki moda` if the text is too long.
```suggestion
        <value>Zobacz pliki modyfikacji</value>
```